boards/shields/nrf7002eb/doc/index.rst

```diff
@@ -0,0 +1,72 @@
+.. _nrf7002eb:
+
+nRF7002 EB
+##########
+
+Overview
+********
+
+The nRF7002 EB is a versatile evaluation kit in the form of a thumbstick shield which connects to
+compatible Nordic host boards, like the Thingy53, using the Nordic edge-connector.
+
+The nRF7002 EB unlocks low-power Wi-Fi 6 capabilities for your host device. It support dual-band Wi-Fi
+2.4GHz and 5GHz, and is based on the nRF7002 SoC.
+Seamlessly connect to Wi-Fi networks and leverage Wi-Fi-based locationing, enabling advanced
+features such as SSID sniffing of local Wi-Fi hubs
+
+.. figure:: nrf7002eb.jpg
+   :alt: nRF7002 EB
+   :align: center
+
+   nRF7002 EB
+
+Requirements
+************
+
+The nRF7002 EB board is designed to fit straight into a Nordic edge-connector and uses SPI as the
+communication interface. Any host board that supports the Nordic edge-connector can be used with
+the nRF7002 EB.
+
+Prerequisites
+-------------
+
+the nRF70 driver requires firmware binary blobs for Wi-Fi operation. Run the command
+below to retrieve those files.
+
+.. code-block:: console
+
+   west update
+   west blobs fetch hal_nordic
+
+Usage
+*****
+
+The shield can be used in any application by setting ``--shield nrf7002eb`` when invoking ``west build``.
+
+Shield Variants
+###############
+
+The nRF7002 EK has a variant which includes the COEX pins. These pins are not be routed to the
+edge-connector on some boards, like earlier revisions of the Thingy53 than v1.0.0.
+
+- ``nrf7002ek``: The default variant.
+- ``nrf7002ek_coex``: Variant which includes the COEX pins.
+
+SR Co-existence
+###############
+
+The nRF7002 EK supports SR co-existence provided the host board supports it. The SR co-existence
```

Comment:
...EB...

There might be more so I trust you will do a full search and replace :)

Reply:
You did catch them all :)